### PR TITLE
Flyout: promote beta dashboard from it

### PR DESCRIPTION
**Pull request by humitos**
Use a dumb URL replacement for now to point users to the new beta dashboard. This is temporal while we are on the transition. Eventually, this will be removed and the correct URL will come from the API once we are migrated.

Related
* #315
* https://github.com/readthedocs/readthedocs-ops/issues/1485

### CODE REVIEW

File: src/flyout.js [hunk 1]
```diff
@@ -190,7 +194,9 @@ export class FlyoutElement extends LitElement {
         <dd>
           <a
             href="${addUtmParameters(
-              this.config.projects.current.urls.downloads,
+              this.config.projects.current.urls.downloads
+                .replace("readthedocs.org", "beta.readthedocs.org")
+                .replace("readthedocs.com", "beta.readthedocs.com"),
```
Comment (agjohnson) [inline src/flyout.js:199]: This view does now currently exist on the new dashboard. See https://github.com/readthedocs/ext-theme/issues/77

It isn't completely useless, but this view is not linked and was never styled:

https://beta.readthedocs.org/projects/docs/downloads/

**Review by agjohnson — COMMENTED** [review]
I think we should just hold off on these links and wait for `app.readthedocs.org` instead of doing this change multiple times.

File: src/flyout.js (hunk 1 above)
Comment (humitos) [reply src/flyout.js:199]: Do you think we should remove the "Downloads" link from the flyout?

**Comment by humitos** [comment]
Next week we are deploying the `app.` domain. I'm marking this as blocked on that for now.

File: src/flyout.js (hunk 1 above)
Comment (agjohnson) [reply src/flyout.js]: Yeah, probably should drop the link to the downloads view. The downloadable formats are already listed in the flyout already, so it does seem redundant already. If users are using the downloads link we can reconsider whether we use the view or not.

Comment (humitos) [reply src/flyout.js]: I've removed this link in https://github.com/readthedocs/addons/pull/322

**Comment by humitos** [comment]
This PR is ready to be merged now that we deployed `app.` domain.

**Review by agjohnson — APPROVED** [review]

**Pull request merged**